.github/workflows/test.yml

```diff
@@ -31,3 +31,17 @@ jobs:
         run: make test-default
       - name: test-prove
         run: make test-prove
+
+  doc-tests:
+    name: doc-tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: doc-tests-${{ runner.os }}-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Why do we want to customize this key?

Reply:
Well, I wasn't sure, but I had understood you mentioned on the call to make it have its own key. If we don't customize it, it would be based on `Cargo.lock`, right? And then it would have the same key as the other job. But not really familiar with this action so not sure.

Reply:
Oh my bad. The action does the correct by default - at least in the sense that we want.

https://github.com/Swatinem/rust-cache?tab=readme-ov-file#cache-details

Its really only when you want to explicitly share cache amongst different jobs that you need to customise it.